pinot-controller/src/main/resources/app/components/Table.tsx

```diff
@@ -562,7 +564,7 @@ export default function CustomizedTables({
                         ) : (
                           <StyledTableCell
                             key={idx}
-                            className={isCellClickable ? classes.isCellClickable : (isSticky ? classes.isSticky : '')}
+                            className={isCellClickable && (!makeOnlyFirstCellClickable || !idx) ? classes.isCellClickable : (isSticky ? classes.isSticky : '')}
```

Comment:
Since `isCellClickable` makes all cells clickable and underlined and that is what may not be wanted in some scenarios. I've added this new `makeOnlyFirstCellClickable` which makes sure that only the first cell is clickable if the flag is set to true

Reply:
what is the usecase here?
I don't see this in the reload segment table

all the other columns are not clickable 
<img width="1387" alt="image" src="https://github.com/user-attachments/assets/5ea03c02-848b-4ef9-a417-69d9b1bdbe69" />

Reply:
Reload status doesn't use the generic table and creates a table of it's own